app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
+
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
```

Comment:
1. Dùng The Safe Navigation Operator (&.) chỗ này được k em
2. Tại sao lại truyền params[:id] chỗ này em nhỉ

Reply:
1. Không được vì sẽ thành &.! không đúng 
2. Khi gửi cho người dùng token để xác thực, ta truyền token đó gắn với symbol :id nên ở đây sẽ lấy là params[:id]